frontend/src/components/Overlay/OverlayNav.tsx

```diff
@@ -0,0 +1,53 @@
+import './OverlayNav.css';
+
+interface OverlayNavProps {
+	totalPages: number;
+	currentPage: number;
+	goToPreviousPage: () => void;
+	goToNextPage: () => void;
+	previousDisabled: boolean;
+	nextDisabled: boolean;
+}
+
+function OverlayNav({
+	totalPages,
+	currentPage,
+	goToPreviousPage,
+	goToNextPage,
+	previousDisabled,
+	nextDisabled,
+}: OverlayNavProps) {
+	const pagination = `Page ${currentPage + 1} of ${totalPages}`;
+
+	return (
+		<nav className="overlay-nav">
+			<div className="previous-container">
+				{!previousDisabled && (
+					<button onClick={goToPreviousPage} className="nav-buttons">
+						<span aria-hidden className="button-icon">
+							◄
+						</span>
+						Previous
+					</button>
+				)}
+			</div>
+			<p>{pagination}</p>
+			<div className="next-container">
+				{!nextDisabled && (
+					<button
+						onClick={goToNextPage}
+						aria-disabled={nextDisabled}
+						className="nav-buttons next"
```

Comment:
This button has className "next" which allows us to apply styles to it, such as `justify-self: end` and `grid-column: 3`. Can do the same for the "previous" button, then there should be no need for those wrapper divs.

Also, `aria-disabled` will always be false here.